components/List/__tests__/__snapshots__/List.test.js.snap

```diff
@@ -4,9 +4,9 @@ exports[` renders List component 1`] = `
 <View>
   <FlatList
     ListHeaderComponent={
-      <Header
-        title="Test Header"
-      />
+      <View>
+        Header Component
```

Comment:
Is this just plain text being passed into the view?

Reply:
This is a mistake and needs to be wrapped inside a Text component. Il update.